src/pages/SingleCell/CustomToolTip.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const CustomToolTip: React.FC = () => {
```

Comment:
I would give this a more specific name (since it's exported).   SingleCellTooltip?